bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -540,7 +540,9 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <!-- ================================================================== -->
 
 <z0param_method>ZengWang2007</z0param_method>
-<z0param_method phys="clm6_0">Meier2022</z0param_method>
+<!-- <z0param_method phys="clm6_0">Meier2022</z0param_method> -->
+<z0param_method use_fates=".true."  phys="clm6_0" >ZengWang2007</z0param_method>
```

Comment:
@ekluzek, in general does the order of options in the namelist_defaults_ctsm.xml matter? I.e. is it important the "use_fates" preceed "phys"?

Reply:
No, the order does NOT matter in terms of getting the right result.

I think the file may follow some conventions on the order, which can make it easier to read. I've done vertical alignment for that sort of thing as well, but we are trying to get away from that sort of thing.

Reply:
do we need this line?
> <z0param_method use_fates=".true."  phys="clm6_0" >ZengWang2007</z0param_method>

The default was already set to ZengWang2007.  It seems like we covered our bases by applying the use_fates tag here: 

> <z0param_method use_fates=".false." phys="clm6_0" >Meier2022</z0param_method>

Reply:
Yes, thanks @rgknox that's a bit cleaner/clearer way to do it. Let the default be set as ZengWang for everything and only overridden for non-FATES and clm6_0 physics.

Reply:
@ekluzek @rgknox this is what I thought too, but I just tested removing this line and the build fails saying that Meier2022 is still being set somehow:
```
ERROR: Command /home/glemieux/Repos/ESCOMP/ctsm/bld/build-namelist failed rc=255
out=
err=ERROR : CLM build-namelist::CLMBuildNamelist::setup_logic_roughness_methods() : z0param_method = Meier2022 and use_fates currently are not compatible.  Please update the z0param_method to ZengWang2007.  See issue #2932 for more information.
```

Reply:
Try it again with adding use_fates to the add_default call for z0param_method, as I suggest above.